src/language-server/utilities/debug.ts

```diff
@@ -1,4 +1,5 @@
-import { Connection } from "src/messages";
+import { LanguageServerNotifications as Notifications } from "../../messages";
+import { Connection } from "vscode-languageserver/node";
```

Comment:
```suggestion
import type { Connection } from "vscode-languageserver/node";
```

Not sure if we are planning to use `import type` where possible in this project or not, but figured I'd spot some of the places that look like type imports 🙂

Reply:
Tbh., this will keep happening a lot. I'll do a cleanup swipe over all of these after we have a few PRs in :)